week2/task6/Program.cs

```diff
@@ -0,0 +1,59 @@
+﻿// 6. Let's get things done
+
+var signaler = new Signaler();
+
+
+signaler.AddTime(new JupiterTime(1, 20));
+signaler.AddTime(new JupiterTime(2, 20));
+signaler.AddTime(new JupiterTime(3, 20));
+signaler.Inform();
+
+
+public class Signaler
+{
+    List<JupiterTime> Timers = new List<JupiterTime>();
+
+
+    public void AddTime(JupiterTime time)
+    {
+        Timers.Add(time);
+    }
+    public void Inform()
+    {
+        if (Timers.Count == 0)
+        {
+            Console.WriteLine("No timers added yet.");
+            return;
+        }
+
+        foreach (var Timer in Timers)
```

Comment:
since `Timer` in the foreach is a variable, it should be defined with lowercase `foreach (var **timer** in Timers)`